README.md

```diff
@@ -345,6 +345,16 @@ the attribute type does not have any special semantic or effect currently.
 As already mentioned, all this configuration parameters can also be adjusted using the
 [`config.js`](https://github.com/telefonicaid/IoT-STH/blob/develop/config.js) file whose contents are self-explanatory.
 
+It is important to note that there is a limitation on the bytes a MongoDB namespace (concatenation of the database name and
```

Comment:
No, no, the included code can be used with any of the 3 supported data models. The hash is generated from the service path (if collection-per-service-path), the concatenation of service path, entityId and entityType (if collection-per-entity) or the concatenation of service path, entityId, entityType and attributeName (if collection-per-attribute). See https://github.com/telefonicaid/IoT-STH/pull/84/files#diff-db503fe45ab607476c902dd760bd0defR203 and how the `collectionName4Events` is obtained ;)